src/lib/utils.ts

```diff
@@ -0,0 +1,9 @@
+export const pluralize = (count: number, singular: string, plural: string): string => {
```

Comment:
A little utility function for pluralizing a word in UI that I use in all of my projects